CONTRIBUTING.md

```diff
@@ -2,12 +2,10 @@
 
 Thank you for your interest in contributing! Join the [**Matterpoll channel**](https://community.mattermost.com/core/channels/matterpoll) on the Mattermost community server for discussion about this plugin.
 
-
```

Comment:
Let's add these back

Reply:
Sure? They won't add more space when in preview mode and triggered some Markdown linting warnings :)
<https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md012---multiple-consecutive-blank-lines>

Reply:
1/5 that it's easier to block md code if there are two new lines before a new section. not a blocker.